c++/include/ir.hxx

```diff
@@ -0,0 +1,603 @@
+/*
+ * IR - Lightweight JIT Compilation Framework
+ * (C++ wrapper for IR library)
+ * Copyright (C) 2023 by IR project.
+ * Authors: Anatol Belski <anbelski@linux.microsoft.com>
+ */
+
+#ifndef IR_HXX
+#define IR_HXX
+
+#include "ir.h"
+#include "ir_builder.h"
```

Comment:
Can we include these files under ``ir::c`` namespace to hide C declarations from C++ users.

```c++
namespace ir {
  namespace c {
#  include "ir.h"
#  include "ir_builder.h"
  }
    const auto consistency_check = ir::c::ir_consistency_check;
```